README.md

```diff
@@ -15,6 +15,61 @@
 3. Blazing Fast: Tachyon's foremost requirement is speed, and not just any speed, but blazing speed! This entails Tachyon delivering exceptional performance on both CPU and GPU platforms.
 4. GPU Interoperability: Tachyon's code is designed to be compatible with both CPU and GPU in the majority of scenarios.
 
+## List of Features
+
+List of symbols:
+
+- :heavy_check_mark: means the feature is currently supported.
+- 🏗️ means that the feature is partially implemented or is under active construction.
+- :x: means that the feature is not currently supported.
+
+### Finite Fields
+
+|            | CPU                | GPU |
+| ---------- | ------------------ | --- |
+| Goldilocks | :heavy_check_mark: | :x: |
+
+### Elliptic Curves
+
+|           | CPU                | GPU                |
+| --------- | ------------------ | ------------------ |
+| bn254     | :heavy_check_mark: | :heavy_check_mark: |
+| bls12-381 | :heavy_check_mark: | :heavy_check_mark: |
+| secp256k1 | :heavy_check_mark: | :heavy_check_mark: |
+
+### Commitment Scheme
+
+|          | CPU                | GPU |
+| -------- | ------------------ | --- |
+| GWC      | :heavy_check_mark: | 🏗️  |
+| SHPlonk  | :heavy_check_mark: | 🏗️  |
+| FRI      | :heavy_check_mark: | :x: |
+| Pedersen | :heavy_check_mark: | :x: |
+
+### Hash
+
+|          | CPU                | GPU |
+| -------- | ------------------ | --- |
+| Poseidon | :heavy_check_mark: | :x: |
+
+### Lookup
+
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
+### Proving Scheme
+
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
+### Frontend
+
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
```

Comment:
I think we can add R1CS here.

Reply:
I am thinking to add circom here after r1cs parser is completed!